keystore/Cargo.toml

```diff
@@ -12,11 +15,11 @@ parking_lot = "0.12.0"
 serde_json = "1.0.79"
 
 # Substrate dependencies
-sc-keystore = { git = "https://github.com/paritytech/polkadot-sdk.git", branch = "master" }
-sp-application-crypto = { git = "https://github.com/paritytech/polkadot-sdk.git", branch = "master" }
-sp-core = { git = "https://github.com/paritytech/polkadot-sdk.git", branch = "master" }
-sp-keyring = { git = "https://github.com/paritytech/polkadot-sdk.git", branch = "master" }
-sp-keystore = { git = "https://github.com/paritytech/polkadot-sdk.git", branch = "master" }
+sc-keystore = { version = "27.0.0" }
+sp-application-crypto = { version = "32.0.0" }
+sp-core = { version = "30.0.0" }
+sp-keyring = { version = "33.0.0" }
+sp-keystore = { version = "0.36.0" }
```

Comment:
This part we only need for the publishing, right? But we want to update the text below in general, is that right?
then this should be two PRs? Resp. only the PR for the text. And the dependencies here would only be adjusted for the branch that is published on crates.io.

Reply:
Opened a separate PR. See https://github.com/scs/substrate-api-client/pull/757